From 0f4d212874c4fbee18b860963d0a5c7bd54dcfd1 Mon Sep 17 00:00:00 2001
From: Michael Webster <miketwebster@gmail.com>
Date: Mon, 31 Jan 2022 08:56:28 -0500
Subject: [PATCH] network: Remove old workaround for openvpn widgets.

I can't reproduce the original issue anymore, and NMACertChooser
is removed in network-manager-openvpn 1.8.34.

Fixes #10616

ref:
6db4961fd3a002182984d0e71ec364c9befdd93b
---
 meson.build                       |  2 --
 panels/network/cc-network-panel.c | 10 ----------
 2 files changed, 12 deletions(-)

diff --git a/meson.build b/meson.build
index f1f213b..c4d7d86 100644
--- a/meson.build
+++ b/meson.build
@@ -56,8 +56,6 @@ else
   libnma= dependency('', required: false)
 endif
 
-config.set('HAVE_NMA_18', libnm.version().version_compare('>=1.8.0'))
-
 if get_option('modemmanager')
   if not get_option('networkmanager')
     error('*** NetworkManager is required by ModemManager ***')
diff --git a/panels/network/cc-network-panel.c b/panels/network/cc-network-panel.c
index a4289de..d1925e4 100644
--- a/panels/network/cc-network-panel.c
+++ b/panels/network/cc-network-panel.c
@@ -47,10 +47,6 @@
 #include <libmm-glib.h>
 #endif
 
-#ifdef HAVE_NMA_18
-#include <nma-cert-chooser.h>
-#endif
-
 CC_PANEL_REGISTER (CcNetworkPanel, cc_network_panel)
 
 #define NETWORK_PANEL_PRIVATE(o) \
@@ -1244,12 +1240,6 @@ cc_network_panel_init (CcNetworkPanel *panel)
                 return;
         }
 
-#ifdef HAVE_NMA_18
-        /* some newer VPN plugins pre-require internal resources from libnma */
-        /* this solution is really ugly, but works clean */
-        gtk_widget_destroy (nma_cert_chooser_new ("dummy", NMA_CERT_CHOOSER_FLAG_NONE));
-#endif
-
         panel->priv->cancellable = g_cancellable_new ();
 
         panel->priv->treeview = GTK_WIDGET (gtk_builder_get_object (panel->priv->builder,
